metrics-facade-samples/src/main/java/com/ringcentral/platform/metrics/samples/reporters/PrometheusMetricsExporterSample.java

```diff
@@ -72,19 +80,40 @@ public static void main(String[] args) throws Exception {
 
         PrometheusSampleMaker sampleMaker = new PrometheusSampleMaker();
 
-        PrometheusInstanceSamplesProvider miSamplesProvider = new PrometheusInstanceSamplesProvider(
-            miSampleSpecProvider,
-            miSampleSpecModsProvider,
-            miSampleMaker,
-            sampleSpecProvider,
-            sampleSpecModsProvider,
-            sampleMaker,
-            registry);
+        PrometheusInstanceSamplesProvider miSamplesProvider = prometheusInstanceSamplesProvider(registry)
+            .instanceSampleSpecProvider(miSampleSpecProvider)
+            .instanceSampleSpecModsProvider(miSampleSpecModsProvider)
+            .instanceSampleMaker(miSampleMaker)
+            .sampleSpecProvider(sampleSpecProvider)
+            .sampleSpecModsProvider(sampleSpecModsProvider)
+            .sampleMaker(sampleMaker)
+            .build();
 
         PrometheusMetricsExporter exporter = new PrometheusMetricsExporter(
             true,
             Locale.ENGLISH,
-            miSamplesProvider);
+            miSamplesProvider,
+            new SimpleCollectorRegistryPrometheusInstanceSamplesProvider(
+                name("defaultRegistry"), // optional prefix
+                new SampleNameFilter.Builder().nameMustNotStartWith("counter").build(), // optional filter
+                sampleName -> !sampleName.endsWith("created"), // optional filter
+                CollectorRegistry.defaultRegistry));
+
+        Counter defaultRegistry_Counter = Counter.build()
+            .name("counter")
+            .help("Counter from defaultRegistry")
+            .register();
+
+        defaultRegistry_Counter.inc(3);
+
+        Summary defaultRegistry_Summary = Summary.build()
+            .name("summary")
+            .help("Summary from defaultRegistry")
+            .register();
+
```

Comment:
It would be nice to add, what output is expected.

Reply:
You can run the sample and get the output via http://localhost:9095. 
I don't see the need for adding the output to the code.

Reply:
I've created the follow-up task to improve the docs accordingly.